### PR TITLE
feat: try to invalidate realpath cache if keypair loading failed

### LINKED ISSUE #1045: PHP realpath cache and Kubernetes secrets / configmap updates
Hello there,

I'm using this bundle in my Symfony application, deployed in production with Kubernetes.

Keypairs are shared between pods and mounted as symlinks. When rotating the keypairs, the symlink are updated, but the PHP cache of realpath is not.  
This leads to errors on `file_get_contents()` and finally, the pods crash.

Here is a complete explanation of the problematic with some solutions: https://pracucci.com/php-realpath-cache-and-kubernetes-secrets-configmap-updates.html

I suggest invalidating the realpath cache if `file_get_contents` failed at first attempt.

Regards,

### PR DESCRIPTION
**Pull request by lobodol**
Closes https://github.com/lexik/LexikJWTAuthenticationBundle/issues/1045

### CODE REVIEW

**Review by chalasr — APPROVED** [review]

**Comment by lobodol** [comment]
Thanks for your approval @chalasr. Do you plan to merge it soon?

File: Services/KeyLoader/AbstractKeyLoader.php [hunk 1]
```diff
@@ -51,7 +51,17 @@ public function getAdditionalPublicKeys(): array
                 throw new \RuntimeException(sprintf('Additional public key "%s" does not exist or is not readable. Did you correctly set the "lexik_jwt_authentication.additional_public_keys" configuration key?', $key));
             }
 
-            $contents[] = is_file($key) ? file_get_contents($key) : $key;
+            if (is_file($key)) {
```
Comment (lobodol) [inline Services/KeyLoader/AbstractKeyLoader.php:56]: The right part of the ternary expression is missing in this refactoring. Add an `else` statement:

```php
if (is_file($key)) {
  //...
} else {
  $contents[] = $key;
}
```

Comment (chalasr) [reply Services/KeyLoader/AbstractKeyLoader.php:56]: Indeed, can you please update the patch?

Comment (lobodol) [reply Services/KeyLoader/AbstractKeyLoader.php:56]: Sure, done.

Comment (lobodol) [reply Services/KeyLoader/AbstractKeyLoader.php:56]: Oh crap, some tests seem broken. This is weird, tests pass on my local.

Comment (chalasr) [reply Services/KeyLoader/AbstractKeyLoader.php:56]: No worries, it's not related to this change :)

**Comment by chalasr** [comment]
Thank you @lobodol.

**Pull request merged**